### PR TITLE
Feature/ecs 354 custom data docs

### PR DESCRIPTION
**Pull request by jaapvanblaaderen**
Documentation for custom data

### CODE REVIEW

File: docs/CUSTOM_DATA.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## Custom channel availability
+
+The custom data will be available as a downloadable zip in the [custom channel](https://helpcenter.woodwing.com/hc/en-us/articles/360040134192--Configuring-Studio-for-publishing-to-a-custom-Publish-Channel) (SNS) publish message. The zip contains the entire contents of the `custom` folder. Alongside the url of the zip file, also an eTag is provided. The eTag changes when the zip file is updated, this typically happens when the component set is updated. The eTag can be used to apply caching on the custom data zip in your application logic that processes the custom channel message.
+
+Example of a partial publish message with custom data:
+
+```json
+{
+   "customData": {
+    "eTag": "\"d66341b18e411576ae3ec9fde350e0c3\"",
```
Comment (wwjhu) [inline docs/CUSTOM_DATA.md:14]: Is it really a quoted string?

File: docs/CUSTOM_DATA.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Custom Data
+
+The component set allows you to optionally store files in the `custom` folder. Files in this folder are typically used to store information on how published articles should be post-processed towards their final destination. For example a script or a mapping file that understands how to convert the components in this specific component set to an article published in a web CMS or in an mobile app. Files can be organized in folders and sub-folders.
+
+## Custom channel availability
+
+The custom data will be available as a downloadable zip in the [custom channel](https://helpcenter.woodwing.com/hc/en-us/articles/360040134192--Configuring-Studio-for-publishing-to-a-custom-Publish-Channel) (SNS) publish message. The zip contains the entire contents of the `custom` folder. Alongside the url of the zip file, also an eTag is provided. The eTag changes when the zip file is updated, this typically happens when the component set is updated. The eTag can be used to apply caching on the custom data zip in your application logic that processes the custom channel message.
```
Comment (wwjhu) [inline docs/CUSTOM_DATA.md:7]:
```suggestion
The custom data will be available as a downloadable zip in the [custom channel](https://helpcenter.woodwing.com/hc/en-us/articles/360040134192--Configuring-Studio-for-publishing-to-a-custom-Publish-Channel) (SNS) publish message. The zip contains the contents of the `custom` folder. Alongside the url of the zip file, an eTag is provided. The eTag changes when the zip file is updated, which typically happens when the component set is updated. The eTag can be used to apply caching on the custom data zip in your application logic that processes the custom channel message.
```

Comment (wwjhu) [inline docs/CUSTOM_DATA.md:7]: Does the zip contain just the contents or does it have `custom` as root?

File: docs/CUSTOM_DATA.md (hunk 1 above)
Comment (jaapvanblaaderen) [reply docs/CUSTOM_DATA.md:14]: Yep: https://tools.ietf.org/html/rfc2616#section-14.19

File: docs/CUSTOM_DATA.md (hunk 2 above)
Comment (jaapvanblaaderen) [reply docs/CUSTOM_DATA.md]: It contains the contents inside the folder, not the `custom` folder itself. Do you think this should be more explicit?

Comment (wwjhu) [reply docs/CUSTOM_DATA.md]: No, I think it's already explicitly mentioning that it contains the contents. I was just cross-checking whether my conclusion was correct.

**Review by ruudjungbacker — APPROVED** [review]
The documentation looks good!

The only thing I wonder is that when you take a look at https://helpcenter.woodwing.com/hc/en-us/articles/360040134192--Configuring-Studio-for-publishing-to-a-custom-Publish-Channel that there is also a custom data field in the channel config. Not sure if that becomes confusing.

**Pull request merged**